app/components/ui/Navigation.tsx

```diff
@@ -1,195 +1,46 @@
-import React, { FormEvent, useState } from "react";
-import { Link, useHistory } from "react-router-dom";
-import cn from "classnames";
-import qs from "qs";
-import { useAppContext, whiteLabel } from "utils";
+import React from "react";
+import { Link } from "react-router-dom";
 import Translate from "./Translate";
 import styles from "./Navigation.module.scss";
 
-const {
-  appImages: { logoSmall },
-  logoLinkDestination,
-  navLogoStyle,
-  showMobileNav,
-  showReportCrisis,
-  siteNavStyle,
-  title,
-} = whiteLabel;
-
 export const Navigation = ({
-  showSearch,
```

Comment:
According to the new design it looks like the site search ends up in the secondary nav so I think we would include it there in some other way, maybe conditionally or as passed as `children` to the `SecondaryNavigationLayout` component -- which will likely have to change to account for the headings, text, and site search. Although, the more I look at it now, the design seems to imply it's not a sitewide search but an in-category search since it's next to the top level category heading? 

![image](https://github.com/Exygy/askdarcel-web/assets/5185/e5b014bb-e454-4f45-8093-f8115c1b6e4e)

Reply:
I asked Kyle: https://www.figma.com/design/iDBeNK7SnZXF8wc3pbPTCY?node-id=2629-16403#855749264